client/src/App.js

```diff
@@ -1,18 +1,23 @@
-import React from "react";
+import React, { useState } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Home from "./pages/Home";
 import Welcome from "./pages/Welcome";
 import List from "./pages/List";
 import GlobalStyles from "./GlobalStyles";
+import Compare from "./pages/Compare";
 
 function App() {
+  const [listData, setListData] = useState("");
   return (
     <>
       <GlobalStyles />
       <Router>
         <Switch>
+          <Route path="/home/:id/compare">
```

Comment:
`/home/:id/compare` is confusing. `:id` looks like it's the `homeId`. What could be `homeId`? 😃
Paths should target resources like in the API routes. If you like to compare different product lists, I could think about this:
`/lists/:id/compare`.

Reply:
Good point. Changed it. I might need to think of renaming home into lists, too.

Reply:
👍 thank you for the input. I resolved your suggestions.